flask_app.py

```diff
@@ -60,53 +60,50 @@ def execute():
     with open(f"sessions/{session}/.stdin", "w", encoding="utf-8") as f:
         f.write(input_list)
 
-    with open(f"sessions/{session}/.stdin", "r", encoding="utf-8") as x:
```

Comment:
Sorry for the ginormous diff here, but Deepsource said to merge the `with`s into a single one. Let me know if it looks prettier the old way

Reply:
This way's nicer :P